huaweicloud/resource_huaweicloud_lb_loadbalancer.go

```diff
@@ -156,10 +163,14 @@ func resourceLoadBalancerV2Create(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	//set tags
+	elbV2Client, err := config.ElbV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating HuaweiCloud elb v2.0 client: %s", err)
+	}
```

Comment:
suggest put those line in head of the function

Reply:
fixed